### DISCUSSION TITLE
What is the best way to convert the opencv mat to the ndarray Array4?

### DISCUSSION THREAD

**Discussion by yuanyan3060** [rust-ndarray/ndarray#1450] [Q&A]

**Comment by nilgoyette** [comment]
I do not know about the "best" way and I do not have the time to test it, but a quick search returns
- https://github.com/twistedfall/opencv-rust/issues/267
- https://github.com/jerry73204/rust-cv-convert/blob/master/cv-convert/src/with_opencv_ndarray.rs
- https://blog.devgenius.io/rust-and-opencv-bb0467bf35ff

You're free to test it.
